compiler/rustc_borrowck/src/polonius/legacy/mod.rs

```diff
@@ -205,3 +210,31 @@ pub(crate) fn emit_drop_facts<'tcx>(
         });
     }
 }
+
+/// Emit facts about the outlives constraints: the `subset` base relation, i.e. not a transitive
+/// closure.
+pub(crate) fn emit_outlives_facts<'tcx>(
+    tcx: TyCtxt<'tcx>,
+    constraints: &MirTypeckRegionConstraints<'tcx>,
+    location_table: &LocationTable,
+    all_facts: &mut Option<AllFacts>,
+) {
+    if let Some(facts) = all_facts {
```

Comment:
Nit: Would it be better to make this and other similar functions take a `&mut AllFacts` and move this some check out of the function (especially now that you're grouping the function calls together)?

Reply:
Actually, I see you've done this, so this is resolved